frame/contracts/src/lib.rs

```diff
@@ -674,9 +614,34 @@ impl<T: Trait> Module<T> {
 		let result = wasm::save_code_raw::<T>(code, &schedule);
 		result.map(|_| ()).map_err(Into::into)
 	}
+
+	/// Determine the address of a contract,
+	///
+	/// This is the address generation function used by contract instantation. Its result
+	/// is only dependend on its inputs. It can therefore be used to reliably predict the
+	/// address of a contract. This is akin to the formular of eth's CRATE2 opcode. There
+	/// is no CREATE equivalent because CREATE2 is strictly more powerful.
```

Comment:
Sure we want ethereum terminology (CREATE2) in docs?

Reply:
Did not think that intensely about that. What is your recommendation?